tests/backend_test/README.md

```diff
@@ -0,0 +1,36 @@
+# Backend Dev Testing
+
+## Description
+
+- main.sh is the parent file that contains all the tests
+- assert.sh is the code used for the base functions needed to test the backend.
```

Comment:
This file is one directory level up ( .. ).  Let's make it clear to the reader that this is a dependency for main.sh.  The way it reads now, the reader may be expecting this file to exist in the `backend_test` folder.

Reply:
Yeah now there is only one main.sh to avoid confusion. Renamed the test file to backend_test.sh